### PR TITLE
added: navigation component

### PR DESCRIPTION
**Pull request by sharmayatendra**
#Navigation
## Open for review

### CODE REVIEW

File: components/Navigation/navigation.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/header.css">
+    <link rel="stylesheet" href="/css/main.css">
+    <link rel="stylesheet" href="navigation.css">
+    <title>Navigation</title>
```
Comment (mohitdhatrak) [inline components/Navigation/navigation.html:11]: why is your navigation.css file empty? why import then

**Review by mohitdhatrak — COMMENTED** [review]
A suggestion, maybe add a little bit more of documentation on the page

File: components/Navigation/navigation.html (hunk 1 above)
Comment (sharmayatendra) [reply components/Navigation/navigation.html:11]: earlier was thinking of to add some styling to it thats why it is there

**Review by xii-vi — APPROVED** [review]
don't commit an empty file.
For now, you can merge.

**Pull request merged**